contrib/babelfishpg_common/src/collation.c

```diff
@@ -917,9 +922,19 @@ lookup_like_ilike_table(Oid opno)
 	/* return invalid oid when not found */
 	if (!found)
 	{
-		like_ilike_info invalid;
+		like_ilike_info invalid = {
```

Comment:
This doesn't look good to me, we don't need to initialize every member of struct every time.